app/assets/stylesheets/scaffolds.scss

```diff
@@ -21,7 +21,9 @@ a:visited {
 
 a:hover {
   color: #fff;
-  background-color: #000; }
+  background-color: none; }
```

Comment:
Hi Ming, I removed this part because these lines were preventing the links to look how I wanted them to look. These lines made the background dark and the links didn't look aesthetical. I believe these lines are the original code that came in the app when doing scaffold, I remember having the same issue in my personal final project and not being able to find where this style was coming from. I am really happy I was able to find how to get rid of this style as It was driving me crazy in my personal project.